styles.css/main.css

```diff
@@ -3,58 +3,72 @@
      /* Applies to the entire body of the HTML document (except where overridden by more specific
 selectors). */
      /* main.css */   
-        
-div{
+html {
+    box-sizing: border-box;
+}
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+dt::after {
+  content: ":";
+}
+
+::selection {
+  background-color: rgba(0,0,0,0.5);
+}
+
+.width-wrap {
   background-color: #18b0ec;
   text-align: center;
-  padding: .5em;
-  width: 960px;
+  padding: 0 2em 2em;
+  max-width: 960px;
   word-wrap: normal;
   border: 1px black;
-  margin: 0px auto;
+  margin: 0px auto 3em;
   min-width: 480px;
   border-left: 2em solid ;
   border-right: 2em solid;
   border-top: 2em;
-  border-bottom; 2em solid;
+  border-bottom: 2em solid;
   
 }
 
 nav{
 
-  border-top: 4px;
-  border-style: solid;
-  border-color: transparent;
+  border-bottom: 1px dotted rgba(0,0,0,0.3);
+  margin-left: 2em;
+  margin-right: 2em;
 
 }
 
 
 nav ul{
-  max-width: 800;
   margin:0;
   padding:0;
-  list-style-type: none;
-  
- 
+  list-style-type: none; 
   }
 
   nav ul li {
-    float: left;
+    display: inline;
     text-decoration:none;
   
 
   }
-  li a {
-    display: block;
+  header nav li a {
```

Comment:
Make sure you're being specific enough in your selectors.  What you had would style EVERY hyperlink in an `<li>` in your web page